frodo.js

```diff
@@ -1,10 +1,10 @@
 /**
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!